examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -123,12 +123,7 @@ public static void main(String[] args) throws Exception {
         }
 
         try {
-            /* Access a service running on the local machine on port 50051 */
-            String user = "world";
-            if (args.length > 0) {
-                user = args[0]; /* Use the arg as the name to greet if provided */
-            }
-            client.greet(user);
+            client.greet(args[0]);
```

Comment:
I would not support the case (args.length == 2) for simplicity.

Reply:
Done.